server/services/sentence-plan/goalService.ts

```diff
@@ -26,8 +27,8 @@ export default class GoalService {
     return restClient.get<Goals>({ path: `/plans/${parentPlanUuid}/goals` })
   }
 
-  async changeGoalOrder(goals: Goal[]) {
+  async changeGoalOrder(goalOrders: GoalOrder[]) {
     const restClient = await this.sentencePlanApiClient.restClient('Reordering goals')
-    return restClient.post<Goal[]>({ path: `/goals/order`, data: goals })
+    return restClient.post({ path: `/goals/order`, data: goalOrders })
```

Comment:
At the backend we accept an unlimited number of GoalOrder objects in the list - is that OK, or will it always be two, and if so should there be some form of validation (either here or in the backend)?